database/seed.js

```diff
@@ -0,0 +1,89 @@
+const db = require('./database.js');
+const faker = require('faker');
+
+let seedProducts = function(n) {
+  for (let i = 0; i < n; i++) {
+    let temp = faker.random.words(3);
+    db.insertProduct(temp, (err, data) => {
+      if (err) {
+        console.log('error in seeding products function: ' + err);
+      }
+    })
+  }
+};
+
+seedProducts(40);
+
+let convertDate = function(str) {
```

Comment:
This function looks good - I was planning on using the moment library to display dates in my module. Do you think doing it this way is easier?

Reply:
Honestly mysql wasn't letting me insert a date in the format faker.js was giving me. It said sql accepts '2011-01-15' this style. So I changed it. When I look back on it though something else might have actually been broken instead of the date. Who knows? Hopefully moment.js works still on the client side or I will have to change this up again